coldfront/core/project/management/commands/projects.py

```diff
@@ -1,3 +1,5 @@
+from decimal import Decimal
```

Comment:
I haven't tested this manually yet, but it'd be valuable to have a couple unit tests checking key assumptions + correctness.

Relevant tests:
- For renewals: `coldfront.core.project.tests.test_views.test_renewal_views.test_allocation_renewal_request_detail_view`
- For a management command: `coldfront.core.billing.tests.test_commands.test_billing_ids`